test/cri-containerd/gmsa.go

```diff
@@ -0,0 +1,48 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+// Generates a gmsa credential spec and places it at 'path'. Dir has to exist, it will not be created by
+// New-CredentialSpec.
+func generateCredSpec(path string) error {
+	output, err := exec.Command(
+		"powershell",
+		"New-CredentialSpec",
+		"-AccountName",
+		gmsaAccount,
+		"-Path",
+		path,
+	).CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to create new credential spec: %s", string(output))
```

Comment:
This should still wrap the actual error received. e.g. `"failed to create new credential spec (output: %s): %v", string(output), err`.